Source/Core/Core.swift

```diff
@@ -840,7 +846,7 @@ extension FormViewController {
             table.contentInset = tableInsets
             table.scrollIndicatorInsets = scrollIndicatorInsets
             if let selectedRow = table.indexPath(for: cell) {
-                table.scrollToRow(at: selectedRow, at: .none, animated: false)
+                table.scrollToRow(at: selectedRow, at: .none, animated: animateScroll)
```

Comment:
I'm not sure of passing true as animated value works in all cases. In particular when we navigate to a row that is not being displayed in the screen. Have you tested it?